tests/testthat/test-survival_plot.R

```diff
@@ -63,10 +64,13 @@ test_that("survival_plot works as expected", {
         x_label = expression(thd[3])
     )
 
-    vdiffr::expect_doppelganger(
-        "survival_plot with wrap and ci",
-        p1
-    )
+    if (is_graph_snapshot_enabled()) {
```

Comment:
Would it be nicer to use one of the conditional skip functions? eg
```suggestion
    skip_if_not(is_graph_snapshot_enabled()) {
```

Reply:
You're right, I thought it could be done for a block.